presto-postgresql/src/test/java/io/prestosql/plugin/postgresql/TestPostgreSqlTypeMapping.java

```diff
@@ -96,6 +119,35 @@ public final void destroy()
         postgreSqlServer.close();
     }
 
+    @BeforeClass
+    public void setUp()
+    {
+        beforeEpoch = LocalDateTime.of(1958, 1, 1, 13, 18, 3, 123_000_000);
```

Comment:
Why not make these final and initialize in the field declaration?

Reply:
Because there are checks. surefire is apparently dramatically unable to properly report initialization failure when it cannot instantiate test classes